### DISCUSSION TITLE
0316. 去除重复字母

### DISCUSSION THREAD

**Discussion by giscus** [itcharge/AlgoNote#116] [Comments]
# 0316. 去除重复字母 | 算法通关手册

[https://algo.itcharge.cn/solutions/0300-0399/remove-duplicate-letters/](https://algo.itcharge.cn/solutions/0300-0399/remove-duplicate-letters/)

**Comment by s1acr** [comment]
and stack[-1] in letter_counts 这个条件是不是有点多余

**Reply by itcharge** [reply]
确实多余了，感谢指正，文中已更正。
